### PR TITLE
Bdu 62 add quorum back in to proposal details

### LINKED ISSUE #788: Base: add back proposal details quorum %

### PR DESCRIPTION
**Pull request by yayay927**
closes #788 

## Checklist
- [x] Ran Linting
- [x] Targeted PR against correct branch.
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote unit tests.
- [x] Added an entry to the `CHANGELOG.md` file.
- [x] Re-reviewed `Files changed` in the Github PR explorer.

### CODE REVIEW

File: src/screens/proposal_details/components/votes_graph/hooks.tsx [hunk 1]
```diff
@@ -64,6 +68,7 @@ export const useVotesGraph = () => {
         R.pathOr('0', ['stakingPool', 0, 'bondedTokens'], data),
         chainConfig.votingPowerTokenUnit,
       ),
+      quorum: numeral(numeral(quorumRaw).format('0.[00]')).value() * 100,
```
Comment (ryuash) [inline src/screens/proposal_details/components/votes_graph/hooks.tsx:71]: Maybe:

```
quorum: Big(quorumRaw).times(100).toFixed(2)
```

**Review by ryuash — CHANGES_REQUESTED** [review]
Please fix mobile and tablet view

image

File: src/screens/proposal_details/components/votes_graph/components/quorum_explanation/index.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Trans from 'next-translate/Trans';
+import { Typography } from '@material-ui/core';
+import { useStyles } from './styles';
+
+const QuorumExplanation = (props) => {
```
Comment (ryuash) [inline src/screens/proposal_details/components/votes_graph/components/quorum_explanation/index.tsx:6]: props should not be `any`

Comment (yayay927) [reply src/screens/proposal_details/components/votes_graph/components/quorum_explanation/index.tsx:6]: updated

File: src/screens/proposal_details/components/votes_graph/hooks.tsx (hunk 1 above)
Comment (yayay927) [reply src/screens/proposal_details/components/votes_graph/hooks.tsx:71]: updated

**Review by ryuash — CHANGES_REQUESTED** [review]
There is an overflow in desktop now
image

**Pull request merged**